hbw/app/javascript/packs/hbw/components/form/select.js.jsx

```diff
@@ -166,11 +164,11 @@ modulejs.define('HBWFormSelect',
           });
       },
 
-      getDefaultValue (isMulti) {
+      getDefaultValue () {
         const variants = this.buildOptions();
 
         if (this.state.value) {
-          const value = (isMulti ? this.state.value : [this.state.value]).map(el => `${el}`);
+          const value = [this.state.value].map(el => `${el}`);
```

Comment:
You don't need to do this anymore I think.

Reply:
I thought `value` should contain only String